packages/transformers/html/src/inline.js

```diff
@@ -49,12 +52,37 @@ export default function extractInlineAssets(
           type = 'js';
         }
 
+        if (!node.attrs) {
+          node.attrs = {};
+        }
+
+        // allow a script/style tag to declare its key
+        if (node.attrs['data-parcel-key']) {
+          parcelKey = node.attrs['data-parcel-key'];
+        }
+
+        // Inform packager to remove type, since CSS and JS are the defaults.
+        // Unless it's application/ld+json
+        if (
+          node.attrs &&
+          (node.tag === 'style' ||
+            (node.attrs.type && SCRIPT_TYPES[node.attrs.type] === 'js'))
+        ) {
+          node.attrs['parcel-should-delete-type'] = 'true';
+        }
+
+        // insert parcelId to allow us to retrieve node during packaging
+        node.attrs['data-parcel-key'] = parcelKey;
+
         parts.push({
           type,
           code: value,
-          inlineHTML: true,
+          uniqueKey: parcelKey,
+          isIsolated: true,
+          isInline: true,
           meta: {
             type: 'tag',
+            inlineKey: parcelKey,
```

Comment:
Do we need the key in two places? Maybe we could only use `uniqueKey` everywhere?

Reply:
The one in meta was a backup in case we found a way to determine uniqueness outside the transformer